### ISSUE TITLE
Remove inline <code> block styles?

### ISSUE THREAD

**Issue by badlydrawnrob** [badlydrawnrob/anki#32]
**TL;DR**

- Use `markdown->html` with `backticks `...` inline code`
- Add a `b`/`strong`, `i`/`em` tag or use the Anki editor as a visual aid

**More info**

Even [Github Markdown](https://stackoverflow.com/questions/23226224/inline-code-syntax-highlighting-in-github-markdown) doesn't allow inline highlighting (but some flavours do, like Pandoc) ...

- [x] <s>Remove inline highlighting?</s>
    + Perhaps a basic code block + pygments is plenty.
    + **At the very least, it could do with simplification (perhaps just `<b>` and `<i>`)**
    + **Inline `<code>` blocks should (generally) be very simple and short**
- [x] **Just use markdown backticks (other than specific inline code fields)**
    + Leave this up to user discretion (can add `<b>` or `<i>` in Anki editor if needed)
- [x] <s>Markdown generally renders all inline tags (`<b>`, `<mark>`, etc) as part of the `<code>` block ...</s>
    + So it won't render anyway, and not sure if there's a workaround

**Comment by badlydrawnrob** [comment]
For **2.0.0**, sticking with basic `code` css and ability to highlight sections with `b`, `strong`, `i`, and `em` tags — not sure if it's strictly "correct" in validation terms, but most syntax highlighters work this way and it's kind of helpful to visually highlight the code related to the question.
